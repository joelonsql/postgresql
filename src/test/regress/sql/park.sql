--
-- Test backend parking functionality
--

-- Test pg_park function exists and can be called
SELECT pg_park();

-- Test that we can still execute queries after pg_park
SELECT 1 as test_after_park;

-- Test parking with different configurations
-- (These would need enable_parking to be on for full testing)