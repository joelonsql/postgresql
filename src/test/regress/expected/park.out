--
-- Test backend parking functionality
--
-- Test pg_park function exists and can be called
SELECT pg_park();
 pg_park 
---------
 f
(1 row)

-- Test that we can still execute queries after pg_park
SELECT 1 as test_after_park;
 test_after_park 
-----------------
               1
(1 row)

-- Test parking with different configurations
-- (These would need enable_parking to be on for full testing)
